src/main/features/certificateOfService/routes/file-upload.ts

```diff
@@ -43,7 +46,7 @@ export default express.Router()
           new DraftService().save(viewDraft, user.bearerToken).then(() => {
             res.redirect(Paths.documentUploadPage.uri)
           })
-        } else if (FileTypes.acceptedMimeTypes().indexOf(file.type) === -1) {
+        } else if (FileTypes.acceptedMimeTypes().indexOf(fileType(buffer).mime) === -1) {
```

Comment:
I have a vague memory that this library doesn't support detecting `doc` and `xls` files, probably okay but it is on your list of accepted file types.
Could you please test this and report back?
REQ

Reply:
you are correct. weird that it passes in my unit tests. will have to look into that once I find a package that works for the above file types

Reply:
if you can't find one and this support is needed then this issue gives a starting point for adding the support:
https://github.com/sindresorhus/file-type/issues/112